README.md

```diff
@@ -78,6 +78,44 @@ The equivalent of this JSON would be a secret in Infisical with the key `CONNNEC
 
 
 ### InfisicalAuthBuilder Setters
-**SetUniversalAuth*()**
+
+#### SetUniversalAuth()
 - `clientId` (string): The client ID of your universal auth machine identity.
 - `clientSecret` (string): The client secret of your universal auth machine identity.
+
+#### SetAzureAuth()
```

Comment:
nit: SetAzureAuthWithToken(identity, token) so that in the future we can have SetAzureAuth(identityId)